README.md

```diff
@@ -17,10 +17,8 @@ A [Workflow Management System](https://en.wikipedia.org/wiki/Workflow_management
 * [Building](#building)
 * [Installing](#installing)
   * [Upgrading from 0.19 to 0.21](#upgrading-from-019-to-021)
-* [Command Line Usage](#command-line-usage)
-  * [run](#run)
-  * [server](#server)
-  * [version](#version)
+* [**NEW** Command Line Usage](http://gatkforums.broadinstitute.org/dsde/discussion/8782/command-line-cromwell/)
```

Comment:
I like putting it into one line. I wanted to keep it in the TOC so that users who may be used to seeing it there, can still find it. Once there are a few more sections moved over, I might make docs on the WDL website into their own section, so users can easily find what they need while the docs are in transition.